packages/flutter_tools/lib/src/aot.dart

```diff
@@ -26,12 +27,13 @@ class AotBuilder {
     @required String mainDartFile,
     bool bitcode = kBitcodeEnabledDefault,
     bool quiet = true,
-    Iterable<DarwinArch> iosBuildArchs = defaultIOSArchs,
+    Iterable<DarwinArch> iosBuildArchs,
     bool reportTimings = false,
   }) async {
     if (platform == null) {
       throwToolExit('No AOT build platform specified');
     }
+    iosBuildArchs ??= defaultArchsForSdk(SdkType.iPhone);
```

Comment:
This class is _almost_ dead: [b/172093077](https://b.corp.google.com/issues/172093077)